src/Orleans/Providers/IProviderRuntime.cs

```diff
@@ -46,6 +48,11 @@ public interface IProviderRuntime
         /// Service provider for dependency injection
         /// </summary>
         IServiceProvider ServiceProvider { get; }
+        
+        /// <summary>
+        /// Gets or sets the invocation interceptor which will be invoked on each request.
+        /// </summary>
+        InvokeInterceptor InvokeInterceptor { get; set; }
```

Comment:
Do we have to add it here? Seems like it just forces unused implementations of the property.